src/turbolinks/controller.coffee

```diff
@@ -217,6 +218,9 @@ class Turbolinks.Controller
     root = @getSetting("root") ? "/"
     new Turbolinks.Location root
 
+  isCacheable: ->
+    @getSetting("cache-control") != 'no-cache'
```

Comment:
Minor style nit: we've been preferring `isnt` over `!=`, and double over single quotes elsewhere in the source.

```coffeescript
shouldCacheSnapshot: ->
  @getSetting("cache-control") isnt "no-cache"
```